### ISSUE TITLE
GCC 13 cstdint include

### ISSUE THREAD

**Issue by thewtex** [SuperElastix/elastix#916]
`main` with GCC 13:

```
/home/matt/src/elastix/Core/Kernel/elxlog.h:1:1: note: ‘uint8_t’ is def
ined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
  +++ |+#include <cstdint>
    1 | /*=============================================================
============
```

**Comment by N-Dekker** [comment]
Thanks @thewtex So you think it's sufficient to just add an `#include <cstdint>` inside elastix/Core/Kernel/elxlog.h, to fix both this issue and issue #915 (enum usage)?

**Comment by thewtex** [comment]
@N-Dekker that will help to fix this issue, but I think #915 is separate.
